internal/apps/ecs/native.go

```diff
@@ -234,6 +237,30 @@ func (e *ecs) updateTaskDefinition(svc *ecssvc.ECS, td *ecssvc.TaskDefinition, s
 		return fmt.Errorf("unable to update service: %w", err)
 	}
 
+	var dtgo *elbv2.DescribeTargetGroupsOutput
+	if e.Unsafe {
+		pterm.Println("WARNING: deployment will be accelerated (unsafe)")
```

Comment:
Let's put this before we start, and make it an actual warning, otherwise it's hard to see (I didn't notice it first)?
Also, let's add more data which params we are using:
```
WARNING: deployment will be accelerated (unsafe):
- Health Check Interval: 5s
- Health Check Timeout: 2s
- Healthy Threshold Count: 2
- Unhealthy Threshold Count: 2
```

Reply:
@AutomationD [![asciicast](https://asciinema.org/a/Qp3v6dORpvWL9zlPpfqcajQVH.svg)](https://asciinema.org/a/Qp3v6dORpvWL9zlPpfqcajQVH)

Is that okay?